TOOLS.md

```diff
@@ -1,19 +1,56 @@
 # List of tools used throughout the project
 
-### npm
+## boltkit
+
+Python code to start a neo4j server.
+
+https://github.com/neo4j-drivers/boltkit/tree/1.3
+
+python -m pip install https://github.com/neo4j-drivers/boltkit@1.3#egg=boltkit
```

Comment:
I feel most of the changes to this file is not needed.

we should rarely include a link to website in docs as 1) it is hard to make sure these links works over time. and 2) it is not that hard to find them with google. So I would suggest to remove all the links added in this doc.

There is no need to mention how to install boltkit and what java version are needed to install for testing as it not specific to this driver. They are instead driver test infrastructure. Writing to here will just add an extra place to maintain the correctly of how to use them. If you find such info is missing, you can add a card in inbox describe what you think it is missing and we will discuss the priority of this job.

I would suggest to remove section boltkit, java version and links that you added in this file.